packages/kmod-6.1-nvidia/kmod-6.1-nvidia.spec

```diff
@@ -102,22 +102,20 @@ install -d %{buildroot}%{tesla_525_libdir}
 install -d %{buildroot}%{_cross_datadir}/nvidia/tesla/module-objects.d
 install -d %{buildroot}%{_cross_factorydir}/nvidia/tesla
 
-sed -e 's|__NVIDIA_VERSION__|%{tesla_525}|' %{S:300} > nvidia-tesla-%{tesla_525}.conf
-install -m 0644 nvidia-tesla-%{tesla_525}.conf %{buildroot}%{_cross_tmpfilesdir}/
+install -m 0644 %{S:300} %{buildroot}%{_cross_tmpfilesdir}/
 sed -e 's|__NVIDIA_MODULES__|%{_cross_datadir}/nvidia/tesla/module-objects.d/|' %{S:301} > \
-  nvidia-tesla-%{tesla_525}.toml
-install -m 0644 nvidia-tesla-%{tesla_525}.toml %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/drivers
+  nvidia-tesla.toml
+install -m 0644 nvidia-tesla.toml %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/drivers
 # Install nvidia-path environment file, will be used as a drop-in for containerd.service since
 # libnvidia-container locates and mounts helper binaries into the containers from either
 # `PATH` or `NVIDIA_PATH`
 sed -e 's|__NVIDIA_BINDIR__|%{_cross_libexecdir}/nvidia/tesla/bin|' %{S:302} > nvidia-path.env
 install -m 0644 nvidia-path.env %{buildroot}%{_cross_factorydir}/nvidia/tesla
-# We need to add `_cross_libdir/tesla_525` to the paths loaded by the ldconfig service
+# We need to add `_cross_libdir` to the paths loaded by the ldconfig service
 # because libnvidia-container uses the `ldcache` file created by the service, to locate and mount the
 # libraries into the containers
-sed -e 's|__LIBDIR__|%{_cross_libdir}|' %{S:303} | sed -e 's|__NVIDIA_VERSION__|%{tesla_525}|' \
-  > nvidia-tesla-%{tesla_525}.conf
-install -m 0644 nvidia-tesla-%{tesla_525}.conf %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/ld.so.conf.d/
+sed -e 's|__LIBDIR__|%{_cross_libdir}|' %{S:303} > nvidia-tesla.conf
+install -m 0644 nvidia-tesla.conf %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/ld.so.conf.d/
```

Comment:
The unrelated reuse of the `nvidia-tesla.conf` file name is a bit unfortunate: It's both the source of the systemd-tmpfiles configuration and (referencing a file by the same name, generated here) the configuration for `ld` copied into place by that tmpfiles configuration. I'd retain some hint of Source300 being the tmpfiles configuration in its file name (e.g. `nvidia-tesla-tmpfiles.conf`) and specify the target file name when installing: `install -m 0644 %{S:300} %{buildroot}%{_cross_tmpfilesdir}/nvidia-tesla.conf` to remove any ambiguity.

Reply:
Great point. I was mulling this around in my head back and forth, but did not come to a good conclusion here. I will adjust.